contracts/core/modules/lib/TokenFlush.sol

```diff
@@ -0,0 +1,184 @@
+/*
+    Copyright 2019 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.5.7;
+
+import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+import { AddressArrayUtils } from "../../../lib/AddressArrayUtils.sol";
+import { ERC20Wrapper } from "../../../lib/ERC20Wrapper.sol";
+import { ISetToken } from "../../interfaces/ISetToken.sol";
+import { ModuleCoreStateV2 } from "./ModuleCoreStateV2.sol";
+
+
+/**
+ * @title TokenFlush
+ * @author Set Protocol
+ *
+ * The TokenFlush contains utility functions to send tokens and base SetTokens from the
+ * Vault or Contract to a specified user address
+ */
+contract TokenFlush is 
```

Comment:
Is this meant to be reused elsewhere? It's named like it's general but it seems highly coupled with the system. Would prefer it to be specific here instead if it's only done so we can test the library methods. `RebalancingSetExchangeTokenChangeHandler` or something

Reply:
We previously had it in `RebalancingSetIssuance` - but I realize that we will be using flushing functions for other modules in the future (e.g. token transformation pipeline). So that's why it's refactored to be generic.

Reply:
sg!